docs/en/test_and_ci/fuzz_tests.md

```diff
@@ -0,0 +1,69 @@
+# Fuzz Tests
+
+Fuzz tests are a specialized form of [unit tests](../test_and_ci/unit_tests.md) that run code against a large number of random inputs.
+This helps to ensure that the code is robust against any input, not just those expected by the developer.
+
+They can be written like unit tests with possible assertions (`EXPECT_EQ`, etc), and have a set of input parameters.
+The fuzzer then tries to find inputs that cause the code to crash (with enabled Address Sanitizer) or trigger an assertion.
```

Comment:
What is an address sanitizer? Can we link to something. Does a user need to do anything special to get this to work?

Reply:
Yes here: https://clang.llvm.org/docs/AddressSanitizer.html
It's enabled automatically.

Reply:
```suggestion
The fuzzer then tries to find inputs that cause the code to crash (with [Address Sanitizer](https://clang.llvm.org/docs/AddressSanitizer.html) enabled automatically) or trigger an assertion.
```